endpoints/openrtb2/amp_auction.go

```diff
@@ -436,6 +442,9 @@ func getExtBidResponse(
 		warnings = make(map[openrtb_ext.BidderName][]openrtb_ext.ExtBidderMessage)
 	}
 	for _, v := range errortypes.WarningOnly(errs) {
+		if errortypes.ReadScope(v) == errortypes.ScopeDebug && !(reqWrapper != nil && reqWrapper.Test == 1) {
+			continue
```

Comment:
Could you add a test to cover this line?

Reply:
added `TestAmpAuctionDebugWarningsOnly`